scripts/index.mjs

```diff
@@ -22,10 +22,11 @@ current = createFeed(feed, current);
 const loadMore = document.querySelector(".load");
 const logout = document.querySelector(".logout");
 
-loadMore.addEventListener("click", (e)=>{
+if(loadMore){
```

Comment:
The load more button also created an issue  when the post response was with fewer than 20 posts.